assets/src/dashboard/components/pageStructure/navigationComponents.js

```diff
@@ -43,72 +42,73 @@ export const Header = styled.h1`
   }
 `;
 
-export const NavButton = styled(Button)`
-  margin-bottom: 0;
-  margin-top: 0;
-  background-color: ${({ theme }) =>
-    theme.DEPRECATED_THEME.colors.foreground.gray12};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.black};
-  border-radius: 4px;
-  border-color: ${({ theme }) =>
-    theme.DEPRECATED_THEME.colors.foreground.gray12};
-  height: 36px;
-
-  &:hover,
-  &:focus,
-  &:focus-within {
-    background-color: ${({ theme }) =>
-      theme.DEPRECATED_THEME.colors.foreground.gray16};
-    color: ${({ theme }) => theme.DEPRECATED_THEME.colors.black};
-  }
-`;
+export const NewStoryButton = styled(Button)(
+  ({ theme }) => css`
+    margin-bottom: 0;
+    margin-top: 0;
+    border: 1px solid ${theme.colors.border.defaultActive};
```

Comment:
2px for now until we sort out how we want to handle needing a 2px wide border on focus so that there's no shift. we'll handle the update in the `Button` for continuity.

Reply:
Let's also add `    &:focus {
      box-shadow: none;
    }` to this button and to the `NavLink` to override the box-shadow that common css is adding through wordpress. 

Eventually we should be able to control this with the global style overrides in the design system but we're in this kinda murky middle ground. It's something we'll have to make sure is fixed before starting in on the editor.